### PR TITLE
React (Virtual) DOM Terminology doc updated

### PR DESCRIPTION
**Pull request by gustavovnicius**
ES6 examples instead of React.createClass

### CODE REVIEW

File: docs/docs/ref-10-glossary.md [hunk 1]
```diff
@@ -89,14 +89,20 @@ These are used as properties of other `ReactElement`s to represent children. Eff
...
+  render() {
     ...
   }
-});
+}
+```
+
+Or you can define it as a function (but it will be a stateless `ReactComponent`):
+```javascript
+let MyComponent = () => {
```
Comment (satya164) [inline docs/docs/ref-10-glossary.md]: Why not `const` instead?

Comment (gustavovnicius) [inline docs/docs/ref-10-glossary.md]: I think thats the habit. Changed it 😄

**Comment by facebook-github-bot-0** [comment]
@gustavovnicius updated the pull request.

File: docs/docs/ref-10-glossary.md [hunk 2]
```diff
@@ -89,14 +89,20 @@ These are used as properties of other `ReactElement`s to represent children. Eff
...
+  render() {
     ...
   }
-});
+}
+```
+
+Or you can define it as a function (but it will be a stateless `ReactComponent`):
+```javascript
+const MyComponent = () => {
```
Comment (jimfb) [inline docs/docs/ref-10-glossary.md]: Let's change this to:

```
function MyComponent() {
  return ...;
}

The syntax is likely more familiar to new users, and has better compatibility across browsers.

**Comment by facebook-github-bot-3** [comment]
@gustavovnicius updated the pull request.

**Comment by gaearon** [comment]
Thanks! We removed this section in new docs because it was quite out of date.
We might re-add it in some form in #8060 so please track progress there.

**Pull request closed without merging**